docker/r_fungi/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM rocker/r-ver:3.5.0
```

Comment:
This Dockerfile is used for the ChromoMap ploting and I hijacked it from another source where it was used in context of a fungi annotation project :) Thus, we could also rename it to something like `r_chromomap` to be more descriptive

Reply:
and: 

```docker: Error response from daemon: pull access denied for microbiomeinformatics/r_fungi, repository does not exist```

Reply:
good catch, I'll fix this.